web/help/GuideUnit1.md

```diff
@@ -16,45 +16,45 @@ CodeWorld logo in the *canvas* on the right side of the screen.
 
     Computers can be very picky, so make sure you have typed *exactly* what
     you see above.  That means everything needs to be spelled correctly, the
-    same letters need to be capitalized, and you need parentheses just where
+    same letters need to be capitalized and you need parentheses just where
     they are.  There should be nothing else in your editor except that one
     line.
 
     When you make a mistake, the computer usually shows you an *error message*,
-    which shows up on the right side of your screen, in a pink shaded window.
+    which shows up on the right side of your screen in a pink shaded window.
     These messages tell you what went wrong.
 
     ![](cw-error-notinscope.png)
 
-    * If you see "not in scope", this means you have misspelled a word, or
+    * If you see "not in scope", this means you have misspelled a word or
       used the wrong capitalization.
     * If you see "parse error", this can mean you've left out or have
       extra punctuation marks or symbols.
 
-    Check your code carefully, and try again.
+    Check your code carefully and try again.
 
-Dissecting your first program
+Dissecting Your First Program
 -----------------------------
 
 If you have done coding before, you may have heard that code is like a recipe
 that tells a computer step-by-step what to do.  In CodeWorld, though, code is
 more like a *dictionary* or *glossary*. It tells the computer what words mean.
 
-Here's the code you just wrote, and what its parts mean.
+Here's the code you just wrote and what its parts mean
 
     | `program`  | `=` | `drawingOf(` | `codeWorldLogo`     | `)` |
     |------------|-----|--------------|---------------------|-----|
     | My program | is  | a drawing of | the CodeWorld logo. |     |
 
 * `program` is the **variable** that you're defining. A variable is a name for
-  something.  In most math, variables are just one letter long, and they stand
+  something.  In most math, variables are just one letter long and they stand
   for numbers.  In CodeWorld, though, variables can name many different types of
-  values: numbers, pictures, colors, text, and even whole programs.  Because
```

Comment:
I read about them just today. I had no idea. I do not remember anything like that from English class, but it's been a really, really long time since I've had English. I've actually taken more German since my last English class and I might have things stuck in my head from it, who knows.

Reply:
fixed